agent/src/beerocks/slave/son_slave_thread.h

```diff
@@ -241,10 +241,8 @@ class slave_thread : public beerocks::socket_thread {
                                        uint8_t authkey[32], uint8_t keywrapkey[16]);
     bool autoconfig_wsc_parse_m2_encrypted_settings(std::shared_ptr<ieee1905_1::tlvWscM2> m2,
                                                     uint8_t authkey[32], uint8_t keywrapkey[16],
-                                                    std::string &ssid, sMacAddr &bssid,
-                                                    WSC::eWscAuth &auth_type,
-                                                    WSC::eWscEncr &encr_type, bool &backhaul,
-                                                    bool &fronthaul, bool &teardown);
+                                                    bool &backhaul, bool &fronthaul, bool &teardown,
```

Comment:
I'm confused, didn't we talk about removing the backhaul fronthaul and teardown booleans since they are part of the configData?